test/SideNavSpec.js

```diff
@@ -46,6 +46,11 @@ describe('<SideNav />', () => {
     assert.isTrue(sideNav.hasClass('side-nav fixed red'));
   });
 
+  it('should render trigger on large screens if `showOnLarge`', () => {
```

Comment:
let's also add some specs to check the `fixed` logic and what happens when there's no `trigger`